### PR TITLE
Create database from diagram

### PR DESCRIPTION
**Pull request by ezraermy**
This pull request creates a database using a given clinic diagram.
The implementation includes the following changes:
- Create multiple tables
- Use primary key & foreign key mechanisms for joining tables.
- Add a diagram for the vet_clinic database.
[vet_clinic_diagram](https://github.com/ezraermy/vet_clinic/assets/35451409/194b7d67-f8af-4b8b-8cc7-e27d8d329973)

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  treatment_id INTEGER NOT NULL,
+  FOREIGN KEY (invoice_id) REFERENCES invoices (id),
+  FOREIGN KEY (treatment_id) REFERENCES treatments (id)
+);
+
+-- create FK indexes
+CREATE INDEX idx_medical_histories_patient_id ON medical_histories (patient_id);
+CREATE INDEX idx_invoices_medical_history_id ON invoices (medical_history_id);
+CREATE INDEX idx_invoice_items_invoice_id ON invoice_items (invoice_id);
+CREATE INDEX idx_invoice_items_treatment_id ON invoice_items (treatment_id);
```
Comment (Ibizugbe) [inline schema_based_on_diagram.sql:59]: Nice job so far, just as required for this milestone, we are to add a foreign key index for each foreign key 😉. you haven't added fk index for the foreign keys associated with the `treatments_medical_histories` table😉. Kindly do that, please.
image

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi Team👋👋, 

tryagain


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅

- [x] Gitflow have been used👍.
- [x] All tables have a primary key set👍.
- [x] The new tables have been be created in `schema_based_on_diagram.sql` file.





## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ezraermy** [comment]
@Ibizugbe Thank you for the feedback.

**Review by julie-ify — APPROVED** [review]
Hi @ezraermy and @getssh,

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ezraermy** [comment]
Thanks @julie-ify we appriciate your feedback.

**Pull request merged**